plugins/sfInstallPlugin/lib/sfInstall.class.php

```diff
@@ -531,4 +533,17 @@ public static function addSymlinks()
 
     return $symlinks;
   }
+
+  /**
+   * Make any additional sql changes that cannot be made via schema.yml.
+   * Needs to be run for new installs and the tools:purge task.
```

Comment:
Should it be called from that task then?

Reply:
sfInstall::loadData() is called from the tools:purge task here:

https://github.com/artefactual/atom/blob/qa/2.5.x/lib/task/tools/purgeTask.class.php#L115